src/db/mapping.cr

```diff
@@ -65,6 +65,10 @@ module DB
     {% end %}
 
     {% for key, value in properties %}
+      {% value[:nilable] = true if value[:type].is_a?(Generic) && value[:type].type_vars.map(&.resolve).includes?(Nil) %}
```

Comment:
The types come through as `Generic` ast nodes represented as `::Union(String, ::Nil)` so it requires quite a song and dance to ascertain if the types are nilable. Is there any way to improve this @asterite?